Dockerfile

```diff
@@ -45,10 +45,13 @@ ARG GIT_COMMIT
 ARG GIT_TREE_STATE
 
 RUN GOOS=${TARGETOS} GOARCH=${TARGETARCH} go build \
+      -trimpath \
+      -ldflags "-w -s" \
```

Comment:
Note: `-s` is fine here, as I expect the chances for us to have the desire to `pprof` this are minimal as opposed to the controller binary.